pyproject.toml

```diff
@@ -40,6 +40,12 @@ exclude_lines = [
     'if __name__ == .__main__.:',
 ]
 
+[tool.pytest.ini_options]
+markers = [
+    "v1_3_0: marks tests for release 1.3.0",
+    "v1_4_0: marks tests for release 1.4.0",
```

Comment:
What is the meaning of these new markers?

Reply:
we can collect and run TCs to generate some validation report for the next release by using the marker to the newly implemented test cases for the new features of the release.